assets/airports/lrop.json

```diff
@@ -0,0 +1,1150 @@
+        "SISGU":   ["N44d22m09.891", "E25d27m52.661"],
+        "SOKRU":   ["N44d59m34.136", "E25d19m49.048"],
+        "SOMOV":   ["N43d42m00.000", "E24d51m00.000"],
+        "SORDU":   ["N44d02m33.097", "E25d26m48.226"],
+        "STJ":     ["N44d55m06.820", "E25d58m37.300"],
+        "TABAV":   ["N44d53m22.808", "E26d58m01.012"],
+        "TEVRO":   ["N44d20m47.081", "E25d30m07.333"],
+        "TIMUR":   ["N43d41m20.000", "E24d14m21.001"],
+        "TOBAG":   ["N44d22m37.978", "E26d42m09.949"],
+        "TOSVI":   ["N44d45m14.010", "E25d09m40.719"],
+        "TUMOP":   ["N44d19m03.599", "E26d17m49.709"],
+        "TUTIX":   ["N44d14m24.422", "E25d31m07.611"],
+        "UBOGU":   ["N44d45m47.001", "E24d40m05.998"],
+        "URELA":   ["N45d29m47.911", "E26d33m39.931"],
+        "USIRO":   ["N44d14m30.397", "E25d45m52.599"],
+        "UTIDI":   ["N44d15m10.031", "E26d14m01.934"],
+        "UTOKI":   ["N44d44m39.242", "E25d51m53.111"],
+        "UVALU":   ["N44d42m38.891", "E25d26m38.864"],
+        "VAMON":   ["N44d23m57.998", "E24d40m46.999"],
+        "VEVIN":   ["N44d42m47.001", "E26d39m06.998"],
+        "VIKBI":   ["N44d15m30.999", "E24d39m20.998"]
+    },
+    "airways": {
+        "M747": ["ENIMA", "SOKRU", "VEVIN", "IDARU", "DIRAL"],
+        "L5":   ["UBOGU", "TOSVI", "NETUL", "RIVOS"],
+        "L601": ["DENAK", "VEVIN"],
+        "P727": ["ENIMA", "ARGES"],
+        "N181": ["FLR", "POLUN", "ELBAV", "TIMUR"],
+        "L622": ["KOMAN", "OBUGA", "FLR", "ENIMA"],
+        "T746": ["VEVIN", "OSTAL", "ARGES"]
+    },
+    "runways": [
+        {
+            "name": ["08R", "26L"],
+            "end": [
+                ["N44d33m52.78", "E26d04m36.29", "314ft"],
+                ["N44d34m04.69", "E26d07m13.54", "314ft"]
+            ],
+            "ils": [true, true],
+            "ils_distance": [30, 30],
+            "loc_maxDist": [28, 28],
+            "glideslope": [2.70, 2.70]
+        },
+        {
+            "name": ["08L", "26R"],
+            "end": [
+                ["N44d34m35.84", "E26d05m02.76", "313ft"],
+                ["N44d34m47.73", "E26d07m39.95", "313ft"]
+            ],
+            "ils": [true, true],
+            "ils_distance": [30, 30],
+            "loc_maxDist": [28, 28],
+            "glideslope": [3.00, 3.00]
+        }
+    ],
+    "arrivalRunway": "08R",
+    "departureRunway": "08L",
+    "sids": {
+        "SOKRU": {
+            "icao": "SOKRU",
+            "name": "Sokru",
+            "suffix": {"08R": "1K", "08L": "1K", "26R": "1M", "26L": "1M"},
+            "rwy": {
+                "08R": [["OP882", "S220-"], "OP890", ["OP891", "A45+"], "GOMDI"],
+                "08L": [["OP881", "A09+|S220-"], "OP890", ["OP891", "A45+"], "GOMDI"],
+                "26R": [["OP663", "S250-"], "UTOKI"],
+                "26L": [["OP664", "S250-"], "UTOKI"]
+            },
+            "body": ["SOKRU"],
+            "exitPoints": {
+                "ENIMA": ["ENIMA"],
```

Comment:
`ENIMA` and `IRLOX` are not exitPoints, the procedure ends at `SOKRU`

Reply:
they are not, indeed, they are in the procedure to avoid having `body` or `exitPoints` as empty arrays. what's the workaround?

Reply:
No workaround is needed, the body can be left empty

Reply:
changes commited